extensions/package.json

```diff
@@ -9,12 +9,17 @@
   },
   "scripts": {
     "postinstall": "node ./postinstall.mjs",
-    "compile": "../node_modules/.bin/tsc -b",
-    "watch": "../node_modules/.bin/tsc -b -w"
+    "compile": "yarn compile:gitpod-web-res && yarn compile:ts",
+    "watch": "npm-run-all watch:ts watch:gitpod-web-res",
+	"compile:ts": "../node_modules/.bin/tsc -b",
+	"watch:ts": "../node_modules/.bin/tsc -b -w",
+	"compile:gitpod-web-res": "yarn --cwd gitpod-web svelte:build",
+	"watch:gitpod-web-res": "yarn --cwd gitpod-web svelte:dev"
```

Comment:
fix formatting

Reply:
I have no idea why the indent is not the same, even though I copy that indent from above...

Reply:
I think you are using tabs and should be spaces